ci/kind/kind-setup.sh

```diff
@@ -206,6 +206,7 @@ function load_images {
       continue
     fi
     echo "loaded image $img"
+    df -h
```

Comment:
Is this added intended or for debugging?

Reply:
These are all debug logs, The fix was not ready for review.